infra/package-lock.json

```diff
@@ -1153,10 +1340,21 @@
         "node": ">= 4.5.0"
       }
     },
+    "node_modules/available-typed-arrays": {
+      "version": "1.0.5",
+      "resolved": "https://registry.npmjs.org/available-typed-arrays/-/available-typed-arrays-1.0.5.tgz",
+      "integrity": "sha512-DMD0KiN46eipeziST1LPP/STfDU0sufISXmjSgvVsoU2tqxctQeASejWcfNtxYKqETM1UxQ8sp2OrSBWpHY6sw==",
+      "engines": {
+        "node": ">= 0.4"
+      },
+      "funding": {
+        "url": "https://github.com/sponsors/ljharb"
+      }
+    },
     "node_modules/aws-cdk-lib": {
-      "version": "2.63.2",
-      "resolved": "https://registry.npmjs.org/aws-cdk-lib/-/aws-cdk-lib-2.63.2.tgz",
-      "integrity": "sha512-Kk9aP79O+9UJ3Bw0wjquMD+RpWPQ/Fa4wUcP5mdpROpoipPXsFFZPTXT8j8kp2oFOZa9OHxacTF06+/M0H/bWQ==",
+      "version": "2.60.0",
```

Comment:
Any particular reason why we need to downgrade `aws-cdk-lib` here?  Shouldn't we be using the latest?

Reply:
Unfortunately the latest version had incompatibilities with `eks-blueprints`. Weird stuff like the `ec2.InstanceType` wouldn't work. I had to downgrade to the version they list.